kik_unofficial/callbacks.py

```diff
@@ -1,5 +1,24 @@
+from typing import Union
 from kik_unofficial.datatypes.xmpp.account import GetMyProfileResponse, GetMutedConvosResponse
-from kik_unofficial.datatypes.xmpp.chatting import *
+from kik_unofficial.datatypes.xmpp.chatting import (
```

Comment:
I don't like this, I prefer `import *` over this long line

Reply:
the thing with * is that it also imports everything thats ever imported into that file, which imo is a bad habit

Reply:
OK, well, that's true. Can you think of a way to avoid the long list while also not importing everything?

Reply:
we could import `chatting `from `kik_unofficial.datatypes.xmpp` and prepend chatting to all the current imports

Reply:
Great idea